test/validators.js

```diff
@@ -2775,6 +2775,28 @@ describe('Validators', () => {
     });
   });
 
+
+  it('should validate imei strings with hyphens', () => {
+    test({
+      validator: 'isIMEI',
+      args: [{ allow_hyphens: true }],
+      valid: [
+        '35-209900-176148-1',
```

Comment:
Can you add test cases that don't have hyphens here too?

Reply:
Was hoping that if `allow_hyphens` is set to true, it means it won't invalidate those without hyphens but will allow both.

Reply:
allow hypens when allowed will only validate those with hyphens

Reply:
Could this be clearly stated on the README. That by default, only supports IMEI without hyphens?

Reply:
updated @profnandaa